.github/workflows/deploy-managed-ema-image.yaml

```diff
@@ -29,37 +28,19 @@ jobs:
       - name: Login to Amazon ECR
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1.6.0
-      - name: ECR (Dev) - Pull Main Image
-        if: ${{ github.event.inputs.deployEnvironment == 'development' }}
+      - name: Set Build Params
         run: |
-          ECR_DEV_IMAGE="${{ steps.login-ecr.outputs.registry }}/${{ github.event.repository.name }}:main"
-          docker pull $ECR_DEV_IMAGE
-          echo "ECR_DEV_IMAGE=$ECR_DEV_IMAGE" >> $GITHUB_ENV
-      - name: ECR (Dev) - Pull Prod Ready Image Tag
-        if: ${{ github.event.inputs.deployEnvironment != 'development' }}
+          export SKIP_FLAGS_NON_UNIT_TESTS="-Dcheckstyle.skip -Dpmd.skip -Dcpd.skip -Dfindbugs.skip -Dspotbugs.skip"
```

Comment:
Are we adding these to prod since it would make things faster and not needed? or some other reason?

Reply:
That shouldn't have been in there. Thanks! I removed it.